doc/rados/operations/stretch-mode.rst

```diff
@@ -8,110 +8,114 @@ Stretch Clusters
+you might need to use the explicit *stretch mode* to ensure data integrity.
 
 We will here consider two standard configurations: a configuration with two
 data centers (or, in clouds, two availability zones), and a configuration with
-three data centers (or, in clouds, three availability zones).
+three data centers.
 
-In the two-site configuration, Ceph expects each of the sites to hold a copy of
-the data, and Ceph also expects there to be a third site that has a tiebreaker
-monitor. This tiebreaker monitor picks a winner if the network connection fails
-and both data centers remain alive.
+In the two-site configuration, Ceph arranges for each site to hold a copy of
+the data, with a third site that has a tiebreaker (arbiter, witness)
+monitor. This tiebreaker monitor picks a winner when a network connection
+between sites fails and both data centers remain alive.
 
-The tiebreaker monitor can be a VM. It can also have high latency relative to
-the two main sites.
+The tiebreaker monitor can be a VM. It can also have higher network latency
+to the two main sites.
 
-The standard Ceph configuration is able to survive MANY network failures or
-data-center failures without ever compromising data availability. If enough
-Ceph servers are brought back following a failure, the cluster *will* recover.
+The standard Ceph configuration is able to survive many network failures or
+data-center failures without compromising data availability. When enough
+cluster components are brought back following a failure, the cluster will recover.
 If you lose a data center but are still able to form a quorum of monitors and
-still have all the data available, Ceph will maintain availability. (This
+still have replicas of all data available, Ceph will maintain availability. This
 assumes that the cluster has enough copies to satisfy the pools' ``min_size``
 configuration option, or (failing that) that the cluster has CRUSH rules in
 place that will cause the cluster to re-replicate the data until the
-``min_size`` configuration option has been met.)
+``min_size`` configuration option has been met.
 
 Stretch Cluster Issues
 ======================
 
-Ceph does not permit the compromise of data integrity and data consistency
+Ceph does not compromise data integrity and data consistency
 under any circumstances. When service is restored after a network failure or a
-loss of Ceph nodes, Ceph will restore itself to a state of normal functioning
-without operator intervention.  
+loss of Ceph nodes, Ceph will return to a state of normal function
+without human intervention.  
 
 Ceph does not permit the compromise of data integrity or data consistency, but
 there are situations in which *data availability* is compromised. These
-situations can occur even though there are enough clusters available to satisfy
-Ceph's consistency and sizing constraints. In some situations, you might
+situations can occur even though there are sufficient replias of data available to satisfy
```

Comment:
s/replias/replicas/

I will deal with this in a follow-up PR in the near future.